changelog/6150.feature.rst

```diff
@@ -0,0 +1,4 @@
+Add ``not_supported_language_list`` to component to be able to define languages that a component can NOT handle.
+
+``WhitespaceTokenizer`` is not able to process languages which are not separated by whitespace. ``WhitespaceTokenizer``
+will throw an error if it is used with Chinese, Japanese, Korean, Thai, and Vietnamese.
```

Comment:
@dakshvar22 Just to be sure: Any other language to add or should I remove any of those languages?

Reply:
I also wasn't sure about it. This link seems to answer our question - https://linguistics.stackexchange.com/questions/6131/is-there-a-long-list-of-languages-whose-writing-systems-dont-use-spaces

Reply:
thanks, so will remove Korean and Vietnamese from the list